src/runtime/mod.rs

```diff
@@ -130,14 +115,36 @@ impl Runtime {
 
 impl Drop for Runtime {
     fn drop(&mut self) {
-        // drop tasks
+        // drop tasks in correct order
         unsafe {
             ManuallyDrop::drop(&mut self.local);
-            ManuallyDrop::drop(&mut self.rt);
+            ManuallyDrop::drop(&mut self.tokio_rt);
         }
     }
 }
 
+fn start_uring_wakes_task(
+    tokio_rt: &tokio::runtime::Runtime,
+    local: &LocalSet,
+    driver: driver::Handle,
+) {
+    let _guard = tokio_rt.enter();
+    let async_driver_handle = AsyncFd::new(driver).unwrap();
+
+    local.spawn_local(drive_uring_wakes(async_driver_handle));
```

Comment:
If I may, I think 'driver' is still being used too much. Everything is a driver. Then nothing is. 😄

Reply:
Yeah, we should probably rename a bunch of the `driver` vars to `driver_handle`.